CHANGES.rst

```diff
@@ -1,3 +1,589 @@
+4.3rc1 (2021-06-18)4.3rc1 (2021-06-18)
+===================
+
+New Features
+------------
+
+astropy.convolution
+^^^^^^^^^^^^^^^^^^^
+
+- Change padding sizes for ``fft_pad`` in ``convolve_fft`` from powers of
+  2 only to scipy-optimized numbers, applied separately to each dimension;
+  yielding some performance gains and avoiding potential large memory
+  impact for certain multi-dimensional inputs. [#11533]
+
+astropy.coordinates
+^^^^^^^^^^^^^^^^^^^
+
+- Adds the ability to create topocentric ``CIRS`` frames. Using these,
+  ``AltAz`` calculations are now accurate down to the milli-arcsecond
+  level. [#10994]
+
+- Adds a direct transformation from ``ICRS`` to ``AltAz`` frames. This
+  provides a modest speedup of approximately 10 percent. [#11079]
+
+- Adds new ``WGS84GeodeticRepresentation``, ``WGS72GeodeticRepresentation``,
+  and ``GRS80GeodeticRepresentation``. These are mostly for use inside
+  ``EarthLocation`` but can also be used to convert between geocentric
+  (cartesian) and different geodetic representations directly. [#11086]
+
+- ``SkyCoord.guess_from_table`` now also searches for differentials in the table.
+  In addition, multiple regex matches can be resolved when they are exact
+  component names, e.g. having both columns “dec” and “pm_dec” no longer errors
+  and will be included in the SkyCoord. [#11417]
+
+- All representations now have a ``transform`` method, which allows them to be
```

Comment:
Some of these changelog entries are unusually long, maybe it's a side effect of towncrier? Overall I would reemphasize that these should be much briefer, keep the details for the what's new.

Reply:
Yeah, a good point. I'm guessing it's the psychology that you don't see a bunch of other bullets that are short and to the point?  So yeah, emphasizing briefer is a good idea.  Maybe we should update the PR guidelines for this or something?